app/models/manageiq/providers/nuage/network_manager/event_parser.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers::Nuage::NetworkManager::EventParser
+  def self.event_to_hash(event, ems_id)
+    {
+      :source     => "NUAGE",
+      :event_type => "#{event['entityType']}_#{event['type'].downcase}",
+      :timestamp  => DateTime.strptime((event["eventReceivedTime"] / 1000).to_s, '%s').to_s,
+      :message    => event.to_hash,
+      :vm_ems_ref => nil,
+      :full_data  => event,
```

Comment:
I guess this will be a hash?
But maybe, just to be sure, when something in qpid changes and they refactor to use a object instead, you'll want to make this `event.to_hash`